src/agents/tools.py

```diff
@@ -38,3 +40,31 @@ def calculator_func(expression: str) -> str:
 
 calculator: BaseTool = tool(calculator_func)
 calculator.name = "Calculator"
+
+
+# Create the embedding function for our project description database
+embeddings = OpenAIEmbeddings()
+
+
+# Format retrieved documents
+def format_contexts(docs):
+    return "\n\n".join(doc.page_content for doc in docs)
+
+
+# Load the stored vector database
+chroma_db = Chroma(persist_directory="./chroma_db", embedding_function=embeddings)
```

Comment:
Will this cause a problem if chroma db was not set up? Would also prefer if running the service without RAG does not create an extra empty db file

Reply:
It may be fine to just move all of this inside the tool function, not sure there’s a high cost to repeatedly re-initializing? Or set it up only the first time it’s called